### PR TITLE
Computing haar features

### PR DESCRIPTION
**Pull request by spark1729**
Implement Haar-like features as discussed in Issuee #1431 .

### CODE REVIEW

**Comment by stefanv** [comment]
No merges on PRs, please

**Comment by ahojnnes** [comment]
Also, please read through this page here: http://scikit-image.org/docs/stable/contribute.html

**Comment by spark1729** [comment]
Sorry, I was a bit confused when checking out the feature branch. Thanks for the headup, will sure follow it from the next time. And I have gone through the contribution workflow.

**Comment by spark1729** [comment]
image

https://travis-ci.org/scikit-image/scikit-image/builds/56050940

 I am facing this problem. It would be great if someone could help me solve this.

**Comment by spark1729** [comment]
image

**Comment by JDWarner** [comment]
@spark1729 I restarted the Travis build now that #1457 is merged - that should sort the flake8 error. It doesn't look like AppVeyor can be restarted. AppVeyor had trouble cloning the repository, likely due to the ongoing DDoS against GitHub, so if we see that error sporadically we can ignore it.

**Comment by blink1073** [comment]
A rebase will be needed to clear the error.

**Comment by blink1073** [comment]
A rebase will be needed to clear the error (I think).

File: skimage/transform/haar_feature.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+def haar_feature(i, x, y, f, s):
+
+    """
+    Haar Features
+    It returns the specific haar feature, evaluated in a 24*24 frame,
+    as a single array.
+
+    Input
```
Comment (emmanuelle) [inline skimage/transform/haar_feature.py]: Please use Numpy documentation standard https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard

with Parameters, Returns, etc. sections

File: skimage/transform/haar_feature.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    ----------
+    Viola, Jones: Robust Real-time Object Detection, IJCV 2001 See pages 1,3.
+
+    Example
+    -------
+    >>> import numpy as np
+    >>> from skimage import data
+    >>> from skimage.transform import integral_image
+    >>> from skimage.transform import haar_feature
+    >>> image=data.coins()
```
Comment (emmanuelle) [inline skimage/transform/haar_feature.py]: pep8: spaces

**Comment by emmanuelle** [comment]
Hi @spark1729 could you please explain what this ``haar_feature`` function is for? As explained in the guidelines, all new functionalities should come with a full example for the gallery. Thanks!

**Pull request closed without merging**